tests/main/core-backlight/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Test system.disable-backlight-service core config option
+
+systems: [ubuntu-core-*]
+
+environment:
+    MASKFILE: /etc/systemd/system/systemd-backlight@.service
+
+execute: |
+    echo "Backlight is not masked by default"
+    systemctl status systemd-backlight@foo | MATCH "Loaded: loaded"
+
+    echo "Check that backlight service can be disabled"
+    snap set core system.disable-backlight-service=true
+
+    systemctl status systemd-backlight@foo | MATCH "Loaded: masked"
+    test -L "$MASKFILE"
```

Comment:
Could you also ensure that readlink shows it's a symbolic link to /dev/null.

Reply:
Done.